README.md

```diff
@@ -119,13 +121,13 @@ This involves comparing to the list in the Google sheet "Cell Line Profiling Sta
 _As the list cannot be parsed, we are not comparing it for now_
 
 
-## For External Users:
+## For External Users <a name="external-users"></a>
 
-> Mind that you need to know about the tools listed above
+> To learn about the tools we use in the pipeline, see [here](#running-pipeline) for a detailed walkthrough
 
 ### Creating your Terra Workspaces:
 
-1. Your first need a terra account with correct billing setup
+1. You first need a terra account with correct billing setup
 2. use the different *_pipeline folders, they contain everything to setup the workspace correctly
   1. use the WDL scripts available in:
     - https://dockstore.org/my-workflows/github.com/broadinstitute/depmap_omics/WGS_pipeline
```

Comment:
I think we would want more information for external users at some point... The goal of this github page is really for external users mostly